Gemfile

```diff
@@ -5,3 +5,13 @@ gemspec
 
 plugin "bundler-inject", "~> 1.1"
 require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundler-inject") rescue nil
+
+minimum_version =
+  case ENV['TEST_RAILS_VERSION']
+  when "7.0"
+    "~>7.0.8"
+  else
+    "~>6.1.4"
+  end
+
+gem "activesupport", minimum_version
```

Comment:
Do we want to use the TEST_RAILS_VERSION in our test?

Don't like how cryptic my example is but...

```ruby
gem "activesupport", "~>#{ENV.fetch("TEST_RAILS_VERSION", "6.1")}.0"
```

I'm not sure that "~>6.1.0" vs "~>6.1.4" will make a great difference.

Reply:
yeah, I had that previously in another PR... since we want to just set minimum versions, and default to 6.1, this was the easiest way to do it.  If you set `TEST_RAILS_VERSION=6.1.6.1` your version (and my original rework, that looks liked yours) didn't work correctly.

This version does what we want minimally... set absolute minimum, prevent arbitrary "MINOR" updates, and also defaults to rails 6.1 instead of relying on the gemspec's maximum version constraint to set the default.  This allows us to use rails 7 here without needing bump the gemspec each time.